tests/unit/type/list.tcl

```diff
@@ -1131,9 +1131,11 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
         assert_equal {} [$rd read]
         # We want to force key deletion to be propagated to the replica 
         # in order to verify it was expiered on the replication stream. 
+        $rd set somekey1 someval1
+        assert_equal {OK} [$rd read]
```

Comment:
FYI, you can do `$rd deferred 0` and `$rd deferred 1` to change a connection between deferring more and normal mode.
maybe the majority of this test will become simpler (calling it `r2` and using deferring mode just for the short period we do the brpop)

Reply:
Let me try that